### DISCUSSION TITLE
Training spacy with a bunch of words instead of sentences.

### DISCUSSION THREAD

**Discussion by Willem770** [explosion/spaCy#7935] [Help: Coding & Implementations] [answered]
I wish to train spacy to find certain words that describe something which differs from a closed list/set of where in different list/ bunch of words it is decrypted differently. I have the data sorted in keys describing them by my data provider. With an example list below it will be c which can be a word or a couple of words and in another list, it will be f also given different things. If I want to train spacy should I merge these words in a string and then train it with that? Will this work since it is not a sentence then. But the model can then maybe be still trained with the words around it because these context needs be given to the model right? 
[a,b,c,d,e] , [g,b,h,f,I].

**Comment by polm** [comment, marked as answer]
You can't really train spaCy to give useful results on single words, no. This is true of basically any NLP system. The closest thing you could do is train a classifier of character sequences. 

Beyond that question I don't really understand your description of your problem. Could you give some example inputs and outputs? Is `c` an input by itself, or do you have `[a, b, c, d, e]` as some kind of sequence? You say they can be "a word or a couple of words", can you give some concrete examples? Either way it does sound like spaCy might not be a good fit for this.
